src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/AnalyzerTest`1.cs

```diff
@@ -125,6 +126,8 @@ public string TestCode
 
         public SolutionState TestState { get; }
 
+        public SolutionState IncrementalChangeTestState { get; }
```

Comment:
💭 Would it make sense to move this into the source generator testing package? It'd lose the ability to test the incrementality of a source generator and test a code fix _in the same test operation_, but I'm not sure that's a significant limitation.

Reply:
I'll try to move it around and see what I can do.

Reply:
I was able to move it down. Let me know what you think.